### ISSUE TITLE
Opening run error 

### ISSUE THREAD

**Issue by bigg02** [KasRoudra2/MaxPhisher#1]
Error in termux after first run
ValueError: could not convert string to float at lines 1507, 1232, 1189

**Comment by andrew2719** [comment]
yeah the same thing. how to resolve that? im using kali linux in VB with bridged adapter. At first it worked and able to get as intended. but when i run again it is throwing the same error 

└─# python3 maxphisher.py

[+] Please wait!
[!] ValueError: could not convert string to float at lines 1507, 1232, 1189  

------error.log----
could not convert string to float: '1.2.2'
could not convert string to float: '1.2.2'
could not convert string to float: '1.2.2'
could not convert string to float: '1.2.2'
HTTPSConnectionPool(host='is.gd', port=443): Max retries exceeded with url: /create.php?format=simple&url=https://fault-turns-thumbs-vpn.trycloudflare.com (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self-signed certificate in certificate chain (_ssl.c:1006)')))
HTTPSConnectionPool(host='api.shrtco.de', port=443): Max retries exceeded with url: /v2/shorten?url=https://fault-turns-thumbs-vpn.trycloudflare.com (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fd346c4cdd0>: Failed to establish a new connection: [Errno -2] Name or service not known'))
could not convert string to float: '1.2.2'
could not convert string to float: '1.2.2'
could not convert string to float: '1.2.2'

**Comment by KasRoudra** [comment]
@andrew2719 @bigg02 Try this [url](https://gitlab.com/KasRoudra/MaxPhisher)

**Comment by andrew2719** [comment]
yeah bro it is working . thank you.

**Comment by sahilbhitale** [comment]
Yeah same problem with me please try to help me .
python3 maxphisher.py
/data/data/com.termux/files/home/MaxPhisher/maxphisher.py:153: SyntaxWarning: invalid escape sequence '\/'
  """
/data/data/com.termux/files/home/MaxPhisher/maxphisher.py:153: SyntaxWarning: invalid escape sequence '\/'
  """
/data/data/com.termux/files/home/MaxPhisher/maxphisher.py:153: SyntaxWarning: invalid escape sequence '\_'
  """
/data/data/com.termux/files/home/MaxPhisher/maxphisher.py:153: SyntaxWarning: invalid escape sequence '\_'
  """

[+] Please wait!
[!] ValueError: could not convert string to float at lines 1507, 1232, 1189

**Comment by lwydady37** [comment]
@sahilbhitale just install it from gitlab here is the link https://gitlab.com/KasRoudra/MaxPhisher

**Comment by Satheesh0924** [comment]
ValueError: could not convert string to float at lines 1507, 1232, 1189
